recipe/meta.yaml

```diff
@@ -10,42 +10,63 @@ source:
   sha256: bd94dd61edae3b23e43e70bbbfbada06e6d2f7fbac292ed1dbcfcb3afcca929a
 
 build:
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  # because of restraint on pandas <2 missing for py312
+  skip: True  # [py>311]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
   number: 0
 
 requirements:
   host:
     - pip
-    - python >=3.6
+    - python
+    - setuptools
     - setuptools-scm
+    - wheel
   run:
+    - python
     - importlib-metadata
-    - python >=3.6
-    - pandas
-    - numpy >=1.15.0
+    - pandas <2
+    - numpy {{ numpy }}
     - statsmodels
     - matplotlib-base
     - pytensor
     - pymc
+    - {{ compiler('cxx') }}
 
 test:
+  source_files:
+    - tests
   imports:
     - causalimpact
   commands:
     - pip check
+    - pytest -v tests  # [not win]
+    # these appear to be the usual pandas-related problem with Int64 vs. Int32 type check
+    - pytest -v -k "not (test_input_w_bad_pre_period or test_input_w_bad_post_period or test_input_w_pre_and_post_periods_having_distinct_classes)" tests  # [win]
   requires:
     - pip
+    - pytest
+    # restrict scipy on osx-64 because of failing imports in tests
```

Comment:
What import failures do you see?

Reply:
I can attach the error.
Admittedly, it seems to occur only for `py311`. [This](https://discourse.pymc.io/t/importerror-cannot-import-name-gaussian-from-scipy-signal/14170) is where I found the hint that it's related to which combination of `arviz` and `scipy` used.

Reply:
[causalimpact-feedstock_osx-error_log.log](https://github.com/user-attachments/files/17991993/causalimpact-feedstock_osx-error_log.log)